scripts/env.sh

```diff
@@ -12,10 +12,5 @@ PARAMETER_FILE=$( cd "$DIR/.." && pwd)/parameters.json
 
 eval $(jq -r '.testconnection | to_entries | map("export \(.key)=\"\(.value|tostring)\"")|.[]' $PARAMETER_FILE)
 
-if [[ -n "$TRAVIS_JOB_ID" ]]; then
-    echo "==> Set the test schema to TRAVIS_JOB_${TRAVIS_JOB_ID}"
-    export SNOWFLAKE_TEST_SCHEMA=TRAVIS_JOB_${TRAVIS_JOB_ID}
-fi
-
```

Comment:
We still need a test schema to be exported, is there somekind of GHA job number that we could use for the different schemas?

Reply:
Do we necessarily need a different test schema for every job id? I currently just added a test schema in the parameter file.